src/main/java/Volkov.java

```diff
@@ -0,0 +1,292 @@
+        }
+        System.out.println(formatResponse(sb.toString()));
+    }
+
+    public static void doMarkCommand(String input) {
+        try {
+            int taskNo = Integer.parseInt(input.substring(MARK_CMD_LENGTH)) - 1;
+            listOfTasks.get(taskNo).markAsDone();
+            String reply = "     Nice! I've marked this task as done:\n"
+                    + "       " + listOfTasks.get(taskNo).toString();
+            System.out.println(formatResponse(reply));
+        } catch (NullPointerException e) {
+            String reply = "     Task number not found, reenter with correct task number:";
+            System.out.println(formatResponse(reply));
+        } catch (NumberFormatException e) {
+            String reply = "     No task number detected, reenter with correct task number:";
+            System.out.println(formatResponse(reply));
+        }
+    }
+
+    public static void doUnmarkCommand(String input) {
+        try {
+            int taskNo = Integer.parseInt(input.substring(UNMARK_CMD_LENGTH)) - 1;
+            listOfTasks.get(taskNo).unmarkAsDone();
+            String reply = "     OK, I've marked this task as not done yet:\n"
+                    + "       " + listOfTasks.get(taskNo).toString();
+            System.out.println(formatResponse(reply));
+        } catch (NullPointerException e) {
+            String reply = "     Task number not found, reenter with correct task number:";
+            System.out.println(formatResponse(reply));
+        } catch (NumberFormatException e) {
+            String reply = "     No task number detected, reenter with correct task number:";
+            System.out.println(formatResponse(reply));
+        }
+    }
+
+    public static void doDeleteCommand(String input) {
+        try {
+            int taskNo = Integer.parseInt(input.substring(DELETE_CMD_LENGTH)) - 1;
+            String reply = "     Noted. I've removed this task::\n"
+                    + "       " + listOfTasks.get(taskNo).toString()
+                    + "     Now you have " + (listOfTasks.size()-1) + " tasks in the list.";
+            System.out.println(formatResponse(reply));
+            listOfTasks.remove(taskNo);
+        } catch (NullPointerException e) {
+            String reply = "     Task number not found, reenter with correct task number:";
+            System.out.println(formatResponse(reply));
+        } catch (NumberFormatException e) {
+            String reply = "     No task number detected, reenter with correct task number:";
+            System.out.println(formatResponse(reply));
+        }
+    }
+
+    public static void doTodoCommand(String input) {
+        try {
+            String taskDesc = input.substring(TODO_CMD_LENGTH);
+            if (taskDesc.trim().isEmpty()) {
+                throw new MissingDescription("MISSING DESCRIPTION, reenter with description");
+            }
+            Todo t = new Todo(taskDesc);
```

Comment:
You might want to name "t" to a more intuitive name so that it increases readability.